### PR TITLE
enable cpp_wrapper/PTQ/QAT inductor accuracy.

### PR DESCRIPTION
**Pull request by zxd1997066**

### CODE REVIEW

File: scripts/modelbench/inductor_quant_acc.py [hunk 1]
```diff
@@ -3,10 +3,11 @@
 import torch
 import torchvision.models as models
 import torch._dynamo as torchdynamo
+import torch._inductor as torchinductor
 import copy
-from torch.ao.quantization.quantize_pt2e import prepare_pt2e, convert_pt2e
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e, convert_pt2e, prepare_qat_pt2e
 import torch.ao.quantization.quantizer.x86_inductor_quantizer as xiq
-from torch.ao.quantization.quantizer.x86_inductor_quantizer import X86InductorQuantizer
+from torch._export import capture_pre_autograd_graph, dynamic_dim
```
Comment (chuanqi129) [inline scripts/modelbench/inductor_quant_acc.py:10]: Seems we don't need `dynamic_dim`

**Comment by chuanqi129** [comment]
@zxd1997066 why we close this PR?
